opentracing-jaxrs2-itest/common/src/main/java/io/opentracing/contrib/jaxrs2/itest/common/rest/InstrumentedRestApplication.java

```diff
@@ -27,7 +27,7 @@ public InstrumentedRestApplication(@Context ServletContext context) {
                 AbstractJettyTest.CLIENT_BUILDER_ATTRIBUTE);
 
         if (serverTracingBuilder == null || clientTracingBuilder == null) {
-            throw new IllegalArgumentException("");
+            throw new IllegalArgumentException();
```

Comment:
Tests are code. You might not care, but if it fails on someone else, you'll save this other person's time if you have a meaningful exception. Either remove this exception and let a NPE explode on the appropriate place, or add a message that will help a future maintainer to have an idea on what's going on.